ecasbot/settings.py

```diff
@@ -78,6 +79,15 @@ def get_cfgpath():
                 return cfgpath
         return '/etc' if os.name == 'posix' else str(os.path.join(os.getenv('APPDATA'), 'ecasbot'))
 
+    @staticmethod
+    def get_logging_level():
+        try:
+            level_name = os.getenv("LOGGING_LEVEL")
```

Comment:
I think `LOGLEVEL` env name will be better.

Reply:
Fixed